package/src/main.rs

```diff
@@ -29,7 +29,7 @@ use crate::utils::fs::{base_name, canonicalize, copy, ensure_directory};
 
 const BINARY: &str = "scie-pants";
 
-const SCIENCE_TAG: &str = "v0.2.2";
+const SCIENCE_TAG: &str = "v0.3.1";
```

Comment:
Another drive-by. This release brings better help when you have lift manifest errors with suggestions for similar names. Release notes:
+ https://github.com/a-scie/lift/blob/main/CHANGES.md#031
+ https://github.com/a-scie/lift/blob/main/CHANGES.md#030